### DISCUSSION TITLE
Dualshock 2 D-Pad issue

### DISCUSSION THREAD

**Discussion by ADMTetanus** [AntiMicroX/antimicrox#784] [Q&A]
The D-Pad seems to be linked to pointing the L Stick in the given directions, rather than being bindable in itself. Is there a way to separate them? is that possible in the scope of this project to be added if it isn't already in there?

![image](https://github.com/AntiMicroX/antimicrox/assets/67378895/7a385ac8-016f-4047-88ea-e5f25853a3f5)
image shows display when D-pad up is held as an example

**Comment by pktiuk** [comment]
Have you tried to use Controller Mapping to fix this?
